packages/ui/components/form/selectimproved/components/SelectProvider.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useContext, useMemo } from "react";
+
+import { ClassNames, Option } from "./type";
+
+interface Store {
+  selectedItems: Option | Option[] | null;
+  handleValueChange: (selected: Option) => void;
+  classNames?: ClassNames;
+}
+
+interface Props {
+  selectedItems: Option | Option[] | null;
+  handleValueChange: (selected: Option) => void;
+  children: JSX.Element;
+  options: {
+    classNames?: ClassNames;
+  };
+}
+
+export const SelectContext = createContext<Store>({
+  selectedItems: null,
+  handleValueChange: (selected) => {
+    console.log("selected:", selected);
```

Comment:
Is this still (gonna be) used?

Reply:
No - this gets overriden on init of store - we just need it to always be defined